pkg/database/migrations.go

```diff
@@ -1968,6 +1968,16 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 					`DELETE FROM users WHERE deleted_at IS NOT NULL`)
 			},
 		},
+		{
+			ID: "00086-AddRealmAutoRotateSetting",
+			Migrate: func(tx *gorm.DB) error {
+				return tx.AutoMigrate(&Realm{}).Error
```

Comment:
if it's just the one bool column, maybe we should just do the standard `ALTER TABLE` dance. I feel like automigrate has screwed us up before

Reply:
Also do we want the column to be NOT NULL?

Reply:
removed auto migrate and added not null.